server/poll/metadata.go

```diff
@@ -5,7 +5,7 @@ type Metadata struct {
 	VotedAnswers           []string `json:"voted_answers"` // VotedAnswers is list of answer that the user with "UserID" have voted for the poll with "PollID"
 	PollID                 string   `json:"poll_id"`
 	UserID                 string   `json:"user_id"`
-	AdminPermission        bool     `json:"admin_permission"` // AdminPermission will be true if the user with "UserID" has admin permission for the poll with "PollID", otherwise false.
+	CanManagePoll          bool     `json:"can_manage_poll"` // CanManagePoll will be true if the user with "UserID" has admin permission for the poll with "PollID", otherwise false.
```

Comment:
```suggestion
	CanManagePoll          bool     `json:"can_manage_poll"` // CanManagePoll will be true if the user with "UserID" can manage the poll with "PollID", otherwise false.
```